### ISSUE TITLE
[Marketing] Write a feature for the Partner Spotlight for the newsletter

### ISSUE THREAD

**Issue by RCheesley** [mautic/hacktoberfest-no-code#69]
This month we have a partner spotlight to feature in the newsletter. Ideally we need to collaborate with the partner and put together some content to feature them in our monthly newsletter which goes out on the first of the month.

This will need some content, some design and a bit of marketer magic to make it interesting and insightful without being overly sales-y.

Let us know if you want to work on this and we'll make the intro to the partner.

Please write the content in this doc: https://docs.google.com/document/d/1kpwJYM00L6wKAW1aLeWRMdK3LiBXX4gV_4xhThJGy4k/edit?usp=sharing

Please ask in #t-marketing if you're stuck.

**Comment by Tobylobah** [comment]
can this be assigned to me @RCheesley

**Comment by RCheesley** [comment]
Sure thing! Please chat in #wg-mautic-newsletter if you'd like further support/direction on this task.

This month's feature will be on Dropsolid. We haven't done a partner feature yet so we'll need to come up with suggestions for the visual design, and also for the actual content. I'll put you in touch with the folks from Dropsolid who will be reviewing the content.

**Comment by RCheesley** [comment]
Yes, we still have some partners we need to feature. We missed it last year but let's get it in this year.
